### PR TITLE
Added skill to tell number of galaxies, stars and planets (belongs to knowledge topic)

### PR DESCRIPTION
**Pull request by AliAyub007**
Fixes issue #75 

Changes: 

Added skill to tell number of galaxies, stars and planets.

Screenshots for the change: 

stars

Etherpad Link: http://dream.susi.ai/p/galaxies

### CODE REVIEW

File: models/general/knowledge/en/number_of_galaxies.txt [hunk 1]
```diff
@@ -0,0 +1,23 @@
+How many galaxies are in this universe| Galaxies in universe | Number of galaxies in universe | Number of galaxies
```
Comment (chashmeetsingh) [inline models/general/knowledge/en/number_of_galaxies.txt]: Make this `How many galaxies are there in this universe| Galaxies in the universe | Number of galaxies in the universe | Number of galaxies`

I suppose same goes for the other two skills as well?

**Review by chashmeetsingh — COMMENTED** [review]
Take a look at the suggestion, I suppose that would be a more appropriate query.

**Review by rishiraj824 — APPROVED** [review]

**Review by chashmeetsingh — APPROVED** [review]
Please squash the commits.

**Review by saurabhjn76 — APPROVED** [review]

**Review by Laughing-Boy — APPROVED** [review]

**Review by isuruAb — APPROVED** [review]

**Comment by hpdang** [comment]
@AliAyub007 check @chashmeetsingh last comment

**Comment by AliAyub007** [comment]
@hpdang done

**Pull request merged**